jaeger-core/src/main/java/io/jaegertracing/internal/clock/SystemClock.java

```diff
@@ -31,7 +31,9 @@
 
   private static int getJavaVersion() {
     val sections = System.getProperty("java.version").split("\\.");
```

Comment:
can we move everything after `System.getProperty("java.version")` into parseJavaVersion() and add a unit test for it running against a few allowed strings?

Reply:
:+1: absolutely

Reply:
@yurishkuro done, thanks for the review